CHANGELOG.md

```diff
@@ -20,10 +20,13 @@ All notable changes to this project will be documented in this file.
 ### Fixed
 
 - An invalid `HdfsCluster` doesn't cause the operator to stop functioning ([#594]).
+- BREAKING: Use distinct ServiceAccounts for the Stacklets, so that multiple Stacklets can be deployed in one namespace. Existing Stacklets will use the newly created ServiceAccounts after restart ([#616]).
```

Comment:
This should be in a new section, since 24.11.0 is already out

Reply:
we are holding the release for this, it'll sneak in

Reply:
We have already tagged 24.11.0. This will be 24.11.1.

Reply:
Tags can change until they are officially communicated ;)

Reply:
But they absolutely shouldn't. Any client that has already "seen" the old tag won't be updated to the new one. This will just cause confusion. Please don't do this.

Reply:
Okay, but what would be the alternative? We are not certifying two releases for OpenShift. Skip 24.11.0 then?

Reply:
Yes, that'd be the way.